### PR TITLE
Max 17

### PR DESCRIPTION
**Pull request by MassimoVilla**
Dear all,
here come my last MS of the Juel-Jensen collection. It is a Maṣḥafa gǝnzat, to which a Maṣḥafa krǝstǝnnā is appended. 
Looking forward to your feedback, comments, and corrections.

### CODE REVIEW

**Comment by MassimoVilla** [comment]
There were some conflicts because this MS has been already covered by Eliana and merged into the Juel-Jensen branch. I have removed all conflicts, but I have left both texts encoded within Condition because they partially overlap (Eliana and I included sometimes different pieces of information). Can I ask you @ElianaDalSasso please to check this part, remove the redundant sentences and merge the two texts into one?
All the remaining parts are now fine

File: OxfordBodleian/Juel-Jensen/BDLaethf33.xml [hunk 1]
```diff
@@ -17,6 +17,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                 <editor key="JG"/>
                 <editor key="MV"/>
                 <editor key="SH"/>
+                <editor key="EDS"/>
```
Comment (gnizla) [inline OxfordBodleian/Juel-Jensen/BDLaethf33.xml]: JK at some point?

Comment (MassimoVilla) [reply OxfordBodleian/Juel-Jensen/BDLaethf33.xml]: Yes, Jacopo, you are right. Shall we add JK to all MSS in one single PR?

Comment (gnizla) [reply OxfordBodleian/Juel-Jensen/BDLaethf33.xml]: Probably a signle PR later on :)

**Review by thea-m — APPROVED** [review]
Thank you Massimo, great work and great addition to Maṣḥafa gənzat encoding as well (I think that we should keep a handy list for the introduction/other occasions of examples where our cataloguing has led to an immediately visible enhancement of BM)

**Review by ElianaDalSasso — CHANGES_REQUESTED** [review]
Thank you Massimo! I tried to merge the texts and answer to your question!

**Review by ElianaDalSasso — APPROVED** [review]
Thank you Massimo!!

**Pull request merged**